### ISSUE TITLE
Downloading HumbleBundle games does not work

### ISSUE THREAD

**Issue by neuromancer** [tkashkin/GameHub#73]
I'm getting 'Forbidden' when I'm trying to install any game from HumbleBundle:

```
[DEBUG 11:19:05.429357] SoupDownloader.vala:179: [SoupDownloader] Downloading to '/home/g/Games/Collections/Humble Bundle/Cat Girl Without Salad/CatGirlWithouSalad_Release_1464898053.zip'
[WARNING 11:19:05.431835] Game.vala:399: Forbidden
```

Resetting the token does not work :cry:

**Comment by tkashkin** [comment]
Is it a regular Humble game or a game from Trove? I have the same problem with the same game from Trove.

Trove links are expiring, so they should be updated.

I don't know if it makes sense to cache Trove games then, anyway it doesn't take too much time to load them again.

**Comment by neuromancer** [comment]
Humble and Trove games are failing :cry:

**Comment by tkashkin** [comment]
Yes, I see humble links have `ttl` too.

Do you have any ideas how to update links? There's a API method for Trove games, but it doesn't work for regular games.

**Comment by neuromancer** [comment]
https://github.com/MayeulC/hb-downloader/issues/3

**Comment by tkashkin** [comment]
Seems to be fixed in 67c77a8, not sure if it'll handle all possible situations correctly though.

**Comment by neuromancer** [comment]
It works as expected. Thanks!
